wolox-books/src/app/screens/book/components/list/list.component.ts

```diff
@@ -13,12 +14,14 @@ import { Book } from '../../book';
 })
 export class ListComponent implements OnInit {
 
-  books: Observable<Book[]>;
+  books: Book[];
 
   search: string;
 
-  constructor(private store: Store<AppState>) {
-    this.books = this.store.select('book');
+  constructor(private store: Store<AppState>, private router: Router) {
+    this.store.select('book').subscribe(data => {
+      this.books = data['books'];
+    });
```

Comment:
```suggestion
    this.store.select('book').subscribe(({ books }) => {
      this.books = books;
    });
```

Reply:
can't do that because we receive an object from the state and we don't know if is empty or not

Reply:
can data property come undefined? 😮

Reply:
already define with type 😄